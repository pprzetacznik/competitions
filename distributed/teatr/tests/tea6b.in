100000000 0 -1 -1 333333 1499293874969838315
