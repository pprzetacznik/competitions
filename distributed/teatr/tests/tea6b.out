2499991157627483
